### DISCUSSION TITLE
It it possible to have nice icons for the indicator?

### DISCUSSION THREAD

**Discussion by brabalan** [meow-edit/meow#601] [Q&A]
I like having a nice indicator of meow’s state in my modeline, and for the moment it’s plain text. It seems it is rendered by `meow--update-indicator` and I’m not sure it can be customized. Has someone tried to change it to icons instead of text?

**Comment by DogLooksGood** [comment]
What kind of icons you want to use?

**Comment by brabalan** [comment]
I thought there were some icons in nerd-fonts, but I no longer find them… I would be happy with the possibility to change the text to something like `<N>` or `<I>` as in this screenshot https://user-images.githubusercontent.com/140797/49694476-b8103880-fbc5-11e8-9c18-91f5e9258333.png
